src/components/template_component/template.sh

```diff
@@ -119,16 +134,22 @@ setup_template() {
                -e 's/SpeciesConc.mode:           '\''time-averaged/SpeciesConc.mode:           '\''instantaneous/g' HISTORY.rc
     fi
 
-    if ! "$isAWS"; then
-	# Load environment with modules for compiling GEOS-Chem Classic
-        source ${GEOSChemEnv}
-    fi
-
     # Remove sample restart file
     rm -f Restarts/GEOSChem.Restart.20190101_0000z.nc4
 
     # Copy template run script
     cp ${InversionPath}/src/geoschem_run_scripts/ch4_run.template .
+
+    if ! "$isAWS"; then
+	if [ ! -f "${InversionPath}/${GEOSChemEnv}" ]; then
```

Comment:
again, maybe just one check in run_imi.sh

Reply:
These lines have been moved to run_imi.sh in https://github.com/geoschem/integrated_methane_inversion/pull/157/commits/9629c549521385b0709e1aa33c1c158082f6ddc7.